### PR TITLE
[14.0] [MIG] account_banking_ach_base

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

File: account_banking_ach_base/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "category": "Banking addons",
+    "depends": ["account_payment_order", "account_banking_mandate"],
+    "data": [
+        "views/account_banking_mandate.xml",
+        "views/account_move.xml",
+        "views/res_bank.xml",
+        "views/res_company.xml",
+        "views/res_partner.xml",
+    ],
+    "external_dependencies": {"python": ["stdnum", "ach"]},
```
Comment (dreispt) [inline account_banking_ach_base/__manifest__.py]: Python library names are now supported as dependencies
```suggestion
    "external_dependencies": {"python": ["python-stdnum", "ach"]},
```

**Review by dreispt — APPROVED** [review]
LGTM

**Comment by dreispt** [comment]
Travis build is failing, @Murtaza-OSI can you fix please?

**Comment by hiren-osi** [comment]
Depends on #76

**Pull request merged**